### PR TITLE
Freezing Python's Dependency

### PR DESCRIPTION
**Pull request by CatCanCreate**
Package versions have been removed so that the user can freeze the latest libraries himself after creating the project.

### CODE REVIEW

File: create_aio_app/template/{{cookiecutter.project_name}}/setup.py [hunk 1]
```diff
@@ -25,24 +24,11 @@ def read_version():
...
-    'aioredis',
-    {%- endif %}
-    # todo: remove test requirements
-    'pytest',
-    'pytest-cov',
-    'pytest-aiohttp',
-]
+def read_requirements():
+    my_file = Path("requirements/production.txt")
+    if my_file.is_file():
```
Comment (Arfey) [inline create_aio_app/template/{{cookiecutter.project_name}}/setup.py]: This condition is not necessarily

File: create_aio_app/template/{{cookiecutter.project_name}}/requirements/development.txt [hunk 2]
```diff
@@ -1,10 +1,10 @@
--e .
+-r production.txt
 
-mypy==0.641
-flake8==3.6.0
-pytest==4.0.1
-pytest-cov==2.6.0
-pytest-aiohttp==0.3.0
+mypy
```
Comment (Arfey) [inline create_aio_app/template/{{cookiecutter.project_name}}/requirements/development.txt]: `Freezing Dependency` its means that you hardcode version of your dependency.
It's good practice :) So please return versions in  `development.txt` and add in `production.txt`

File: create_aio_app/template/{{cookiecutter.project_name}}/setup.py [hunk 3]
```diff
@@ -25,24 +24,11 @@ def read_version():
...
-    {%- if cookiecutter.use_redis == 'y' %}
-    'aioredis',
-    {%- endif %}
-    # todo: remove test requirements
-    'pytest',
-    'pytest-cov',
-    'pytest-aiohttp',
-]
+def read_requirements():
+    my_file = Path("requirements/production.txt")
```
Comment (Arfey) [inline create_aio_app/template/{{cookiecutter.project_name}}/setup.py]: The better way for using `Path` it's declaring the `parent` path and use that in a file. The example u can see in this file.
https://github.com/aio-libs/create-aio-app/blob/master/setup.py#L9

```python
parent = pathlib.Path(__file__).parent

def function(f: str) -> str:
    file_path = parent / f

    with open(file_path) as f:
        return f.read().strip()
```

Comment (Arfey) [reply create_aio_app/template/{{cookiecutter.project_name}}/setup.py]: also will be better if u use annotation for your function

**Comment by Arfey** [comment]
https://github.com/aio-libs/create-aio-app/issues/49

**Comment by CatCanCreate** [comment]
@Arfey Why are you using https://github.com/Arfey/aiohttp-debugtoolbar?

File: create_aio_app/template/{{cookiecutter.project_name}}/requirements/development.txt (hunk 2 above)
Comment (CatCanCreate) [reply create_aio_app/template/{{cookiecutter.project_name}}/requirements/development.txt]: @Arfey 
At work, we use the  skeleton packages without versions, when starting a new project, we freeze the latest versions of packages.
Or do you have code for specific versions?

**Comment by Arfey** [comment]
@CatCanCreate can u fix the tests?

**Comment by Arfey** [comment]
Good job. Thanks.

**Comment by asvetlov** [comment]
I have a simple question.
Do you plan to publish a new `create-aio-app` on PyPI after every dependency will publish a bumped version?

**Comment by asvetlov** [comment]
Than the template created by *a fresh `create-aio-app` install* will start using outdated libraries very soon.
I don't know if that is a big problem but it looks not accurate at least.

**Comment by Arfey** [comment]
We have similar problem with aiohttp demos. Authors didn't freezings dependencies and after some time examples don't run after quickly start because aiohttp remove / changed some features or some conflicts with dependencies. So, I think it is better to use old dependencies, but with a working template.

Maybe once per month we will create new release. Also, I think that good idea it's create make command for update file with dependencies as pyup bot but localy 😀.

**Comment by asvetlov** [comment]
Ok

**Comment by Arfey** [comment]
https://github.com/aio-libs/create-aio-app/issues/51 the issue is waiting for its hero)

**Pull request merged**